spacyServer/spacyServer.py

```diff
@@ -29,21 +31,30 @@ def parse(question):
 			'entType': token.ent_type_,
 			'depType': token.dep_,
 			'depParent': token.head.idx,
+            'depChildren': [],
 		}
 		serializedTokens[token.idx] = serializedToken
 	return toArray(serializedTokens)
 
 def toArray(tokenDict):
 	tokenArray = []
 	keys = sorted(tokenDict.keys())
-	print(keys, file=sys.stderr, flush=True)
+	# print(keys, file=sys.stderr, flush=True)
```

Comment:
I’d prefer deleted lines over commented out. those are easy enough to add again later if desired